src/index.html

```diff
@@ -1,13 +1,321 @@
+        </a>
+
+        <a class="category categories__category" href="#">
+          <div class="pictures category__pictures">
+            <div class="pictures__image-wrapper pictures__image-wrapper--big">
+              <img
+                class="pictures__image pictures__image--show--right"
+                src="./images/photos/speakers-1.jpg"
+                alt="speakers image 1"
+              >
+            </div>
+
+            <div class="pictures__image-wrapper">
+              <img
+                class="pictures__image"
+                src="./images/photos/speakers-2.jpg"
+                alt="speakers image 2"
+              >
+            </div>
+          </div>
+
+          <h3 class="category__name">Speakers</h3>
+        </a>
+
+        <a class="category categories__category" href="#">
+          <div class="pictures category__pictures">
+            <div class="pictures__image-wrapper">
+              <img
+                class="pictures__image"
+                src="./images/photos/audio-sunglasses-1.jpg"
+                alt="audio sunglasses image 1"
+              >
+            </div>
+
+            <div class="pictures__image-wrapper pictures__image-wrapper--big">
+              <img
+                class="pictures__image pictures__image--show--right"
+                src="./images/photos/audio-sunglasses-2.jpg"
+                alt="audio sunglasses image 2"
+              >
+            </div>
+          </div>
+
+          <h3 class="category__name">Audio sunglasses</h3>
+        </a>
+
+        <a class="button categories__button" href="#">View all</a>
+      </section>
+
+      <section class="offer" id="how-to-buy">
+        <h2 class="section-title offer__section-title">Why buy direct from Bose</h2>
+
+        <ul class="offer__benefits">
+          <li class="offer__benefit">Free 2-day shipping and returns</li>
+          <li class="offer__benefit">90-day risk-free trial</li>
+          <li class="offer__benefit">World class customer service</li>
+          <li class="offer__benefit">My Bose account management</li>
+        </ul>
+
+        <p class="offer__description">
+          A great product is more than what’s in the box.
+          It’s a promise of premium performance, world-class support,
+          and everything you expect from a trusted brand.
+          It’s just one of many reasons why you’ll shop with confidence on Bose.com.
+        </p>
+      </section>
+
+      <section class="contact" id="contacts">
+        <h2 class="section-title contact__section-title">Contact us</h2>
+
+        <form class="message contact__message" id="message">
+          <div class="message__item">
+            <label class="message__label" for="name">Name</label>
+
+            <input
+              class="message__input"
+              id="name"
+              name="name"
+              type="text"
+              placeholder="Name"
+              pattern="[A-Za-z]"
```

Comment:
Good job! But we can improve form validation here:

This pattern only allows single-letter input. If you want to allow multiple letters, you can modify the pattern or use + to allow one or more occurrences:
[A-Za-z]+

Also, we can improve email validation with something like this:
pattern="[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}"